airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/AbstractPostgresSourceDatatypeTest.java

```diff
@@ -634,23 +635,6 @@ protected void addTimeWithTimeZoneTest() {
     }
   }
 
-  protected void addTimestampWithInfinityValuesTest() {
```

Comment:
removed the test because 1) it was doing the wrong thing and 2) we added a lot more tests in the original test block